app/models/sync.rb

```diff
@@ -82,7 +83,7 @@ def parents
   end
 
   def parent_blocker
-    parents.not_succeeded.order(position: :asc).first
+    parents.not_succeeded_or_skipped.order(position: :asc).first
```

Comment:
I don't think this was being used anywhere but the UI, but it was still wrong in that the UI would show a skipped transaction as being a blocker